R/02-plotExport.R

```diff
@@ -178,6 +218,92 @@ getFileExt <- function(exportType, isTimeSeries, typeOfSeries, isCollection = FA
   if (typeOfSeries == "onlyGif") return(".gif") else return(".zip")
 }
 
+exportMapRFiles <- function(file, plotFun, Model, input) {
+  withProgress(message = "Generating series ...", value = 0, {
+    times <- seq(input$minTime, input$maxTime, by = abs(input$intTime))
+
+    # create all file names to be put into a zip
+    figFileNames <- sapply(times,
+                           function(i) {
+                             paste0("data","/",
+                                    gsub(" ", "", input$`mapr-group`),"/",
+                                    gsub(" ", "", input$`mapr-variable`),"/",
+                                    gsub(" ", "", input$`mapr-measure`),"/",
+                                    i,".png")
```

Comment:
`png` is indeed the only supported file type from the file types that can by selected by the user. The other supported file type in MapR is `nc`. Therefore, and also because the export zipm file is intended to be uploaded to mapR only and not to be used without the app or something, I thought it might be useful to ignore the file type that was selected. Are you fine with it or should I still add anything? Of course I could change file type to png if mapr is selected. However, I wanted to keep the complexity low. Another option would be to display a note or something. What do you think?

Reply:
Ok, I think than it is good to make a note for the user, e.g. "MapR only supports .png export. Ignoring 'Filetype' input."

![image](https://github.com/user-attachments/assets/0746e276-c686-4d16-9c17-1a193c689fb9)

Reply:
I have added the helptext